### DISCUSSION TITLE
Stupid forward/reverse video movement

### DISCUSSION THREAD

**Discussion by shanislav** [jellyfin/jellyfin-androidtv#2911] [General]
Hi
I just finished switching from plex to jellyfin.

I have chromecast with google tv (4k)

I noticed an unpleasant behavior when jumping forward by 30 seconds.
I press the right button on the TV controller, the video stops, and if I immediately press the select button, the video starts from the beginning. So, every time I move forward, I have to wait a few seconds and then confirm it with the select button.
Through the web browser, this function works without problems, I press the button to the right and the video immediately moves forward without me having to confirm anything.

Am I setting something wrong or is it a feature/bug of the android tv application?

Thank you for your reply.

**Comment by ManfredRichthofen** [comment]
I don't know if this is relevant to you but i had this same issue when running jellyfin behind cloudflare.

**Reply by seamuslowry** [reply]
How did you fix it?

**Reply by ManfredRichthofen** [reply]
I just turned off the proxy(orange cloud) on the domain but if you need it for some reason you could probably bypass the videos using a page rule
